virttest/utils_misc.py

```diff
@@ -84,6 +84,8 @@
 
 
 ARCH = platform.machine()
+PARTITION_TABLE_TYPE_MBR = "msdos"
+PARTITION_TABLE_TYPE_GPT = "gpt"
```

Comment:
they are defined in utils_disk, and utils_disk  is imported in line 74.
So use utils_disk.PARTITION_TABLE_TYPE_MBR and utils_disk.PARTITION_TABLE_TYPE_GPT are OK

Reply:
updated, thanks